### PR TITLE
Auto generate testgrid tab names for release dashboards

### PR DESCRIPTION
**Pull request by krzyzacy**
kinda works? also the re-dump formatted the config a bit.

/area testgrid
/assign @michelle192837 @cjwagner @amwat

### CODE REVIEW

**Comment by krzyzacy** [comment]
/hold
let me clean up some unneeded jobs first

**Comment by krzyzacy** [comment]
(found some legit testgrid config dups, sent another PR to fix)

File: testgrid/config.yaml [hunk 1]
```diff
@@ -3816,330 +3816,330 @@ dashboards:
...
+  - name: gke-device-plugin-gpu-1.11
     test_group_name: ci-kubernetes-e2e-gke-device-plugin-gpu-stable1
-  - name: gke-device-plugin-gpu-p100-1.10
+  - name: gke-device-plugin-gpu-p100-1.11
     test_group_name: ci-kubernetes-e2e-gke-device-plugin-gpu-p100-stable1
 
 - name: sig-release-1.10-blocking
   dashboard_tab:
-  - name: build-1.10
+  - name: build-1.11
```
Comment (cjwagner) [inline testgrid/config.yaml]: If we're going to bump the dashboards like this we'll also need to bump the dashboard name.

**Comment by krzyzacy** [comment]
I think this PR is ok-ish now

I realized kubeadm and conformance jobs also need to be manually created, we can address that as follow ups

**Pull request merged**